### PR TITLE
Add scikit-bio tools

### PR DESCRIPTION
**Pull request by blankenberg**
currently containing a beta diversity tool, able to calculate distances including weighted and unweighted unifrac.

### CODE REVIEW

File: tools/scikit-bio/scikit_bio_diversity_beta_diversity.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+# Reports a beta diversity matrix for tabular input file
+# using scikit-bio
+# Daniel Blankenberg
+
+
+import sys
+import os
+import optparse
```
Comment (bgruening) [inline tools/scikit-bio/scikit_bio_diversity_beta_diversity.py]: Just as a side note, as soon as we assume conda/docker dependencies we can use argparse as we do not need to support python 2.6. any more.

File: tools/scikit-bio/scikit_bio_diversity_beta_diversity.xml [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+        using scikit-bio
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <expand macro="stdio" />
+    <expand macro="version_command" />
+    <command><![CDATA[
+        ##python "${run_scikit_bio}"
```
Comment (bgruening) [inline tools/scikit-bio/scikit_bio_diversity_beta_diversity.xml]: left over from configfile?

**Comment by bgruening** [comment]
Tests pass but you have some pep8 errors:

```
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:9:1: F401 'os' imported but unused
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:41:40: E226 missing whitespace around arithmetic operator
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:44:1: W293 blank line contains whitespace
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:52:40: E226 missing whitespace around arithmetic operator
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:81:9: E265 block comment should start with '# '
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:88:12: E225 missing whitespace around operator
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:88:24: E701 multiple statements on one line (colon)
./tools/scikit-bio/scikit_bio_diversity_beta_diversity.py:88:36: W292 no newline at end of file
```

**Comment by bgruening** [comment]
@blankenberg thanks a lot!

**Comment by blankenberg** [comment]
Thank you for the review!

**Pull request merged**